### PR TITLE
implement modal

### PR DESCRIPTION
**Pull request by jgfidelis**

### CODE REVIEW

File: react/index.tsx [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+  buttonLabel,
+  buttonClass,
+  blockClass,
+}) => {
+  const [isOpen, setOpen] = useState(false)
+  const onClick = useCallback(() => setOpen(true), [])
+  const onClose = useCallback(() => setOpen(false), [])
+
+  const buttonClasses = classNames(
+    `${generateBlockClass(
```
Comment (Klynger) [inline react/index.tsx]: If you're using `classNames` you don't have use string interpolation.

File: react/index.tsx [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+      styles.button,
+      blockClass
+    )} t-action pointer pv3 nv3 ph4 nh4 c-action-primary`,
+    {
+      [`${buttonClass}`]: !!buttonClass,
```
Comment (Klynger) [inline react/index.tsx]: You can just pass `buttonClass` to `classNames` and if `buttonClass`  is falsy, it won't be applied.

**Review by Klynger — CHANGES_REQUESTED** [review]

File: manifest.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{
+  "vendor": "vtex",
+  "name": "modal",
+  "version": "0.0.1",
+  "title": "Modal",
+  "description": "Modal component",
+  "defaultLocale": "pt-BR",
```
Comment (ericreis) [inline manifest.json]: Isn’t it better to use `en-US` as default locale?

**Review by ericreis — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+### Added
+
+- **Component** Create the VTEX Store Component Modal_
```
Comment (klzns) [inline CHANGELOG.md]:
```suggestion
- **Component** Create the VTEX Store Component Modal
```

File: manifest.json (hunk 3 above)
Comment (klzns) [reply manifest.json]: That's not used, can be removed

File: react/.babelrc [hunk 5]
```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["@babel/preset-env", "react-app"]
```
Comment (klzns) [inline react/.babelrc]: Remove

File: .gitignore [hunk 6]
```diff
@@ -0,0 +1,45 @@
+### SublimeText ###
```
Comment (brunojdo) [inline .gitignore:1]: Add README with the proper documentation.

**Review by brunojdo — CHANGES_REQUESTED** [review]

File: manifest.json [hunk 7]
```diff
@@ -0,0 +1,20 @@
+{
+  "vendor": "vtex",
+  "name": "modal",
+  "version": "0.0.1",
```
Comment (brunojdo) [inline manifest.json]: 0.0.0

releasy will bump to 0.0.1

**Review by Klynger — APPROVED** [review]

File: .gitignore (hunk 6 above)
Comment (jgfidelis) [reply .gitignore:1]: readme added

File: react/.eslintrc [hunk 8]
```diff
@@ -0,0 +1,19 @@
+{
+  "extends": "vtex",
+  "env": {
+    "es6": true,
+    "jest": true,
+    "node": true
+  },
+  "rules": {
+    "prettier/prettier": "off",
```
Comment (lbebber) [inline react/.eslintrc:9]: where do these eslint rules come from?

have you followed this recipe? https://docs.google.com/document/d/1ePi5Mn3pe1dOorKrMtC-MclBDiIDimVzf0Uj61W8lA4

I think it's a good standard to follow

**Review by klzns — APPROVED** [review]

**Review by lbebber — APPROVED** [review]

**Pull request merged**